misk/src/test/kotlin/misk/web/interceptors/TracingInterceptorTest.kt

```diff
@@ -40,25 +53,30 @@ class TracingInterceptorTest {
   fun nothingIfNotInstalled() {
     val injector = Guice.createInjector()
 
-    val tracingInterceptorFactory: TracingInterceptor.Factory = injector.getInstance(TracingInterceptor.Factory::class.java)
+    val tracingInterceptorFactory: TracingInterceptor.Factory =
+        injector.getInstance(TracingInterceptor.Factory::class.java)
 
     assertThat(tracingInterceptorFactory.create(TracingTestAction::call.asAction())).isNull()
   }
-}
 
-internal class TracingTestAction : WebAction {
-  @Get("/trace")
-  fun call(): Response<String> {
-    return Response("success")
+  internal class TracingTestAction : WebAction {
+    @Get("/trace")
+    fun call(): Response<String> {
+      return Response("success")
+    }
   }
-}
 
-class MockTracingBackendModule : KAbstractModule() {
-  override fun configure() {
-    bind(Tracer::class.java).to(MockTracer::class.java).asEagerSingleton()
+  internal class TerminalInterceptor(val status: Int) : NetworkInterceptor {
+    override fun intercept(chain: NetworkChain): Response<*>? = Response("foo", statusCode = status)
   }
-}
 
-class NoopModule : KAbstractModule() {
-  override fun configure() {}
-}
+  class MockTracingBackendModule : KAbstractModule() {
+    override fun configure() {
+      bind(Tracer::class.java).to(MockTracer::class.java).asEagerSingleton()
+    }
+  }
+
+  class NoopModule : KAbstractModule() {
+    override fun configure() {}
```

Comment:
supernit: I think you want `= Unit`. I don’t like `{}` because it’s an extra special case in our style rules